k8s/namespaces/ia/ia-case-documents-api/demo.yaml

```diff
@@ -2,8 +2,11 @@ apiVersion: helm.fluxcd.io/v1
 kind: HelmRelease
 metadata:
   name: ia-case-documents-api
+  namespace: ia
 spec:
   values:
+    global:
+      environment: demo
     java:
       image: hmctspublic.azurecr.io/ia/case-documents-api:prod-0a6b3cb-20220308182047 #{"$imagepolicy": "flux-system:ia-case-documents-api"}
```

Comment:
image duplication - can you remove from demo.yaml?

Reply:
The image tag has been removed now. thanks